find-address/src/main/java/com/esri/arcgisruntime/sample/findaddress/MainActivity.java

```diff
@@ -238,6 +228,39 @@ private void showCallout(MotionEvent motionEvent) {
   }
 
   /**
+   * Shows the Graphic's attributes as a Callout.
+   *
+   * @param graphic containing attributes
+   */
+  private void showCallout(final Graphic graphic) {
+    // create a TextView for the Callout
+    TextView calloutContent = new TextView(getApplicationContext());
+    calloutContent.setTextColor(Color.BLACK);
+    // set the text of the Callout to graphic's attributes
+    calloutContent.setText(graphic.getAttributes().get("PlaceName").toString() + "\n"
+        + graphic.getAttributes().get("StAddr").toString());
+    // get Callout
+    mCallout = mMapView.getCallout();
+    // set Callout options: animateCallout: true, recenterMap: false, animateRecenter: false
+    mCallout.setShowOptions(new Callout.ShowOptions(true, false, false));
+    mCallout.setContent(calloutContent);
+    // set the leader position and show the callout
+    mCallout.setLocation(graphic.computeCalloutLocation(graphic.getGeometry().getExtent().getCenter(), mMapView));
```

Comment:
Attach the `GeoElement` to the `Callout`.  This will allow for the callout leader position to follow the symbol offset above: 

```java
// set the leader position and show the callout
Point calloutLocation = graphic.computeCalloutLocation(graphic.getGeometry().getExtent().getCenter(), mMapView);
mCallout.setGeoElement(graphic, calloutLocation);
```

![attach](https://user-images.githubusercontent.com/1308001/30132166-5595510c-9304-11e7-9847-dea1172f4102.png)